spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/java/io/awspring/cloud/sqs/sample/SpringSqsListenMultipleQueues.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2013-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.awspring.cloud.sqs.sample;
+
+import io.awspring.cloud.sqs.annotation.SqsListener;
+import io.awspring.cloud.sqs.operations.SqsTemplate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import software.amazon.awssdk.services.sqs.model.Message;
+
+import java.util.UUID;
+
+@SpringBootApplication
```

Comment:
Having multiple main classes breaks the build. Can we have this sample working as a part of the other main class?

Reply:
Hi @maciejwalkowiak, thank you for you review.

I think that is not necessary to add `@Order` for ApplicationRunner bean.

> Interface used to indicate that a bean should run when it is contained within a [SpringApplication](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/SpringApplication.html). Multiple [ApplicationRunner](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/ApplicationRunner.html) beans can be defined within the same application context and can be ordered using the [Ordered](https://docs.spring.io/spring-framework/docs/6.0.7/javadoc-api/org/springframework/core/Ordered.html) interface or [@Order](https://docs.spring.io/spring-framework/docs/6.0.7/javadoc-api/org/springframework/core/annotation/Order.html) annotation.

What do you think?

Reply:
Only if its relevant for the sample. If sample can work without `@Order` then its better without.

Once you removed `@SpringBootApplication` i think this class is not picked at all right? It must have `@Configuration`